README.md

```diff
@@ -238,6 +239,12 @@ minima:
      - username: jekyll2
        instance: example.com
 
+    gitlab:
+     - username: jekyll
+       instance: example.com
+     - username: jekyll2
+       instance: example.com
```

Comment:
Do we intend to support an array of instances or a single string username..??

Reply:
It's the latter.